README.md

```diff
@@ -1,121 +0,0 @@
-# formatted-stream
-
-A universal wrapper around file format parsers.  
-Currently supports in- and output in CSV, JSON, and XLSX format.
-
-## Usage
-
-Install formatted-stream:
-
-```sh
-npm install formatted-stream --save
-```
-
-Include and use in your code:
-
-#### ES5
-
-```js
-const fs = require('fs'),
-  formattedStream = require('formatted-stream').default,
-  parser = formattedStream.from('json'),
-  writer = formattedStream.to('csv');
-
-parser.pipe(writer);
-writer.pipe(fs.createWriteStream('out.csv'));
-fs.createReadStream('in.json').pipe(parser);
-```
-
-#### ES6
-
-```js
-import fs from 'fs';
-import formattedStream from 'formatted-stream';
-
-const parser = formattedStream.from('json'),
-  writer = formattedStream.to('csv');
-
-parser.pipe(writer);
-writer.pipe(fs.createWriteStream('out.csv'));
-fs.createReadStream('in.json').pipe(parser);
-```
-
-## API Documentation
-
-### .from(format, options)
-
-```js
-const parser = formattedStream.from('csv', {
-  transform: function(data) { return data; },
-  csv: {headers: false},
-  json: [true]
```

Comment:
The idea was to show that you can set different options at the same time, but I suppose it's pretty pointless and might be more confusing than helpful in this particular example.